docs/source/fast_training.rst

```diff
@@ -42,45 +42,26 @@ Must use an int if using an IterableDataset.
     # check every 100 train batches (ie: for IterableDatasets or fixed frequency)
     trainer = Trainer(val_check_interval=100)
 
-Use training data subset
+Use data subset for training, validation and test
 ------------------------
```

Comment:
```suggestion
Use data subset for training, validation and test
-------------------------------------------------
```
This will make the docs build pass